gcp/modules/gcp-stackdriver-alerting/main.tf

```diff
@@ -0,0 +1,49 @@
+terraform {
+  backend "gcs" {}
+}
+
+variable "domain_name" {}
+variable "project_id" {}
+variable "nonce" {}
+variable "serviceaccount_key" {}
+
+resource "template_dir" "resources" {
+  source_dir      = "${path.cwd}/resources"
+  destination_dir = "${path.cwd}/resources_rendered"
+
+  vars {
+    project_id  = "${var.project_id}"
+    domain_name = "${var.domain_name}"
+  }
+}
+
+resource "null_resource" "apply_stackdriver_alerting" {
+  depends_on = ["template_dir.resources"]
+
+  triggers = {
+    nonce = "${var.nonce}"
```

Comment:
It is possible, but we would need to save rendered templates in `destination_dir` locally (using another docker volume probably), otherwise resource re-creation will be triggered on every run. I personally think it does not worth the effort.